src/components/Icon/icons/colors.js

```diff
@@ -0,0 +1,12 @@
+const iconColorsMap = {
+  facebook: '#3B579D',
+  googleplus: '#DB4437',
+  instagram: '#0A0B09',
+  linkedin: '#0077B5',
+  reddit: '#FF4301',
+  twitch: '#6441A4',
+  twitter: '#1DA1F2',
+  youtube: 'red',
```

Comment:
Is youtube really just 'red'? If so, awesome.

Reply:
according to our current icon!